src/pudl_rmi/connect_deprish_to_ferc1.py

```diff
@@ -70,92 +90,77 @@ class InputsCompiler():
         columns (values)
     """
 
-    def __init__(self, file_path_mul, file_path_steam_ferc1,
-                 file_path_ferc1_eia, file_path_deprish_eia, pudl_out):
+    def __init__(
+        self,
+        plant_parts_eia,
+        deprish_eia,
+        ferc1_to_eia
+    ):
         """
         Initialize input manager for connecting depreciation to FERC1.
 
         Args:
-            file_path_mul (path-like): file path to the EIA master unit list
-            file_path_steam_ferc1 (path-like): file path to the compiled FERC1
-                steam table
-            file_path_ferc1_eia (path-like): file path to the table connecting
-                FERC1 steam records to the master unit list
+            file_path_deprish (path-like): path to the excel workbook which
+               contains depreciation data.
+            file_path_ferc1_eia (path-like): file path to the pickled table
+                connecting FERC1 steam records to the master unit list
             file_path_deprish_eia (path-like): file path to the table
                 connecting the depreciation records to the EIA master unit list
+            file_path_training (path-like): file path to the CSV of training
+                data for the FERC1 to EIA connection.
```

Comment:
These args don't seem to match the method call signature. Am I missing something?

Reply:
oh no this is a good catch. this was left over from the first iteration of this coordinating object. I've converted them all to be the dataframes instead of the paths.